### ISSUE TITLE
The path param should be optional

### ISSUE THREAD

**Issue by monolithed** [bcherny/json-schema-to-typescript#63]
```ts
import { compile } from 'json-schema-to-typescript';
compile(scheme); // error
```

```ts
import { compile } from 'json-schema-to-typescript';
compile(scheme, undefined); // ok
```

```ts
export declare function compile(schema: JSONSchema, path: string | undefined, settings?: Settings): string;
```

It's because the path param should be optional:

```ts
export declare function compile(schema: JSONSchema, path?: string, settings?: Settings): string;
```

I guess it's just a typo )

PS: could you export the `JSONSchema` interface?

**Comment by bcherny** [comment]
PRs are welcome!

**Comment by bcherny** [comment]
Fixed in 2391463ea6f9d17b85f6e92717e333c386dd0001

**Comment by bcherny** [comment]
Published 3.1.1
